core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -364,17 +363,17 @@ final case class App(id: AppId,
                      customEnvironmentVariables: Map[String, String],
                      descriptorPath: Option[Uri],
                      extraArgs: List[String]) {
-  def getProxyUrls(project: GoogleProject, proxyUrlBase: String): Map[ServiceName, URL] =
+  def getProxyUrls(project: GoogleProject, proxyUrlBase: String): Map[ServiceName, URL] = {
```

Comment:
I can remove the surrounding `{ }` if you want. I added them during development to break the statement apart, but then inlined everything again at the end and forgot that I had added the `{ }`.

Reply:
I think scalafmt will auto remove these...can you run `sbt scalafmtAll`..CI will fail if it's not run